src/graph/executor/admin/ShowMetaLeaderExecutor.cpp

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "graph/executor/admin/ShowMetaLeaderExecutor.h"
+
+#include <thrift/lib/cpp/util/EnumUtils.h>
+
+#include "graph/context/QueryContext.h"
+#include "graph/planner/plan/Admin.h"
+#include "graph/util/ScopedTimer.h"
+
+namespace nebula {
+namespace graph {
+
+folly::Future<Status> ShowMetaLeaderExecutor::execute() {
+  SCOPED_TIMER(&execTime_);
+  auto metaLeader = qctx()->getMetaClient()->getMetaLeader();
```

Comment:
How about record a time of last hb succeed, when `now - last > threshold`, just return some error.